@here/harp-vectortile-datasource/lib/VectorTileDataSource.ts

```diff
@@ -421,7 +429,7 @@ export class VectorTileDataSource extends TileDataSource {
     setEnableElevationOverlay(enable: boolean) {
         if (this.m_decoderOptions.enableElevationOverlay !== enable) {
             this.m_decoderOptions.enableElevationOverlay = enable;
-            this.configureDecoder(undefined, undefined, undefined, {
+            this.configureDecoder(undefined, undefined, undefined, undefined, undefined, {
```

Comment:
`            this.configureDecoder(undefined, undefined, undefined, undefined, undefined, `
way too many `undefine`s. By just looking at this call it will be impossible to understand what are those paramters

Reply:
same here